src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1731,7 +1731,7 @@ private SolutionState WithAnalyzerConfigDocumentState(AnalyzerConfigDocumentStat
                 return this;
             }
 
-            return this.ForkProject(newProject, new CompilationTranslationAction.ReplaceAllSyntaxTreesAction(newProject));
+            return this.ForkProject(newProject);
```

Comment:
@jasonmalinowski i believe this new line is correct (and that tests validate this sufficiently).  There's no reason to do the replace-all step here as we have already decided earlier which documents to keep (or not) the syntax trees for.  So the compilation naturally just gets updated with the right syntax trees.